ack/readers/mytarget/reader.py

```diff
@@ -182,10 +187,15 @@ def __get_response(self, name_content: str, offset=0) -> Dict[str, Any]:
         """
         parameters = self.__generate_params_dict(name_content, offset=offset)
         request = self.__create_request(name_content, parameters)
-        resp = requests.get(**request).json()
-        if "items" not in resp.keys():
-            raise MissingItemsInResponse("Can't retrieve any item from this response")
-        return resp
+        resp = requests.get(**request)
+        if resp.status_code != HTTP_STATUS_OK:
```

Comment:
Are you sure we never get another 2XX code here ? I'm wondering if getting a 202 for example could occur and then raise an exception while it's not really an error. Maybe need to check the API doc.

Reply:
The document of the API does not explain wether or not we can get another 2XX code. For our use case, we only got 200.